app/controllers/api/v1/keywords_controller.rb

```diff
@@ -11,7 +11,11 @@ def index
       end
 
       def create
-        if csv_form.save(params[:file])
+        saved_keywords = csv_form.save(params[:file])
+        if saved_keywords
+          # TODO: Create background job for all keywords
```

Comment:
```suggestion
          # TODO: Perform "later" and for "all" keywords
```

If you call `perform_now`, it won't run async and will expose your request to timeout.
https://api.rubyonrails.org/classes/ActiveJob/Execution.html#method-i-perform_now

In the final version we need to use [perform_later](https://api.rubyonrails.org/classes/ActiveJob/Enqueuing/ClassMethods.html#method-i-perform_later) instead ;-)